### PR TITLE
Changed the pulse count and event duraction script.

### PR DESCRIPTION
**Pull request by Peterandresen12**

### CODE REVIEW

**Review by MortenHolmRep — COMMENTED** [review]
Add bit type files to .gitignore

**Comment by asogaard** [comment]
What are these binary files you're trying to ignore? 🙂

**Comment by MortenHolmRep** [comment]
> What are these binary files you're trying to ignore? 🙂

The plotting script took some time to run, due to fetching via SQLite query. So peter wanted to download the numpy array for a quicker sampling of the dataset.
So in essence, the binary files contain IceCube data, which should not be on here.

**Comment by asogaard** [comment]
> > What are these binary files you're trying to ignore? 🙂
> 
> The plotting script took some time to run, due to fetching via SQLite query. So peter wanted to download the numpy array for a quicker sampling of the dataset. Pickle generate binary files when it stores data.
> So in essence, the binary files contain IceCube data, which should not be on here.

Wouldn't it make sense, then, to save them as .npy or .npz files, and just put these extensions in the gitignore-file? I'm a little worried about excluding everything and then "unexcluding" directories and files with extensions.

**Comment by Peterandresen12** [comment]
Yes, makes sense, I have changed it such that git ignores .pickle files instead :-)

**Review by MortenHolmRep — CHANGES_REQUESTED** [review]
wandb stuff shouldn't be included

File: .gitignore [hunk 1]
```diff
@@ -135,4 +135,10 @@ dmypy.json
...
 # report types
-*.out
\ No newline at end of file
+*.out
+
+# Pickle types
+*.pickle
+
+# Logs
+wandb/
```
Comment (MortenHolmRep) [inline .gitignore:144]: try `wandb/*`

**Review by asogaard — APPROVED** [review]
Hi @Peterandresen12,

I am fine with the changes made to the common files. I would suggest you remove the wandb file included (`moon_pointing_analysis/modelling/wandb/wandb/latest-run`) before merging, though, but since it only affects your own area, that's entirely up to you. :)

File: .gitignore (hunk 1 above)
Comment (Peterandresen12) [reply .gitignore]: Fixed

**Review by MortenHolmRep — APPROVED** [review]
let's gooooo

**Pull request merged**